spec/mailers/collections_mailer_spec.rb

```diff
@@ -9,6 +9,7 @@
   let(:work) { build_stubbed(:work, collection:) }
   let(:work_version) { build_stubbed(:work_version, work:) }
   let(:a_user) { build_stubbed(:user, name: 'Al Dente', first_name: 'Fred') }
+  let(:mailer_host) { 'www.example.com' }
```

Comment:
I suspect these changes needed making because of the related removed line in `config/application.rb`